src/api-reference/expense/expense-report/v4.reports.markdown

```diff
@@ -577,30 +593,6 @@ curl --location --request GET https://integration.api.concursolutions.com/expens
 ]
 ```
 
-#### Exception Response
-
-```json
-{
-  "timestamp": "2023-05-09T15:14:35.296Z",
-  "httpStatus": "string",
-  "errorMessage": "string",
-  "errorId": "string",
-  "validationErrors": [
-    {
-      "source": "string",
-      "message": "string",
-      "id": "string"
-    }
-  ],
-  "customResponseData": {
-    "additionalProp1": "string",
-    "additionalProp2": "string",
-    "additionalProp3": "string"
-  },
-  "path": "string"
-}
-```
-
```

Comment:
Removed this exception response, I put this here wayyy back when I first did this externalization and its unnecessary.